README.md

```diff
@@ -10,57 +10,66 @@ can then perform whatever actions it wants.
 `snare` requires three command-line arguments to be specified:
 
 ```
-Usage: snare -p <port> -r <repo scripts dir> -s <secret>
+Usage: snare -p <port> -r <repo-scripts-dir> -s <secrets-path>
 ```
 
 where:
 
  * `<port>` is a port number (e.g. 4567)
- * `<repo scripts dir>` is the directory where programs for each repository
-   targeted by the webhook are stored. For a repository `repo` owned by `user`
-   the command `<repo scripts dir>/<user>_<repo> <event> <path to GitHub JSON>`
-   will be run.
- * `<secret>` is the GitHub secret which guarantees that hooks are coming from
-   your GitHub repository and not a malfeasant.
+ * `<repo-scripts-dir>` is the directory where the per-repo programs are
+   stored. For a repository `repo` owned by `user` the command
+   `<repo-scripts-dir>/<user>_<repo> <event> <path to GitHub JSON>` will be
```

Comment:
This is a nitpick, but I wonder if if `<repo-scripts-dir>/user/repo` would be better.

Why?
`<repo-scripts-dir>/simon_123_flibble`

Is this?
 - user: simon, repo: 123_flibble
 - user: simon_123, repo: flibble

If both of those repos existed, the path would clash.

Reply:
Good point.

Reply:
Resolved in https://github.com/softdevteam/snare/pull/1/commits/0b0aa351cf4006c5e2c25d187ee1f46f22afef60